_posts/2017-08-18-comment-verifier-l-orthographe-de-vos-docs-depuis-travis-ci.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Comment vérifier l'orthographe de vos docs depuis Travis CI?
+excerpt: "Nous allons montrer dans cet article comment vérifier l'orthographe automatiquement dans vos documents markdown, modifiés dans vos pull requests, simplement avec Aspell et Travis CI"
+excerpt: "We will show you how to check the spelling in your markdown documents, changed in your pull requests, very easily using Aspell and Travis CI"
```

Comment:
I think this `excerpt` entry should be removed as you already have one (in french) before